desc/objectives/_qs.py

```diff
@@ -178,15 +184,17 @@ def compute(self, *args, **kwargs):
             Quasi-symmetry flux function error at each node (T^3).
 
         """
-        params = self._parse_args(*args, **kwargs)
+        params, constants = self._parse_args(*args, **kwargs)
+        if constants is None:
+            constants = self.constants
         data = compute_fun(
             self._data_keys,
             params=params,
-            transforms=self._transforms,
-            profiles=self._profiles,
+            transforms=constants["transforms"],
+            profiles=constants["profiles"],
         )
-        B_mn = self._matrix @ data["|B|_mn"]
-        return B_mn[self._idx]
+        B_mn = constants["matrix"] @ data["|B|_mn"]
+        return B_mn[constants["idx"]]
```

Comment:
are we basically making everything with not a small constant size, a constant

Reply:
That was the idea. In theory it also allows you to call an objective function and override the default constants, so that in the future we could break some parts out into standalone functions that would be compiled ahead of time, if jax ever gets that working